codalab/server/auth.py

```diff
@@ -66,4 +66,6 @@ def _make_token_request(self, data):
         except urllib.error.HTTPError as e:
             if e.code == 401:
                 return None
-            raise
+            if e.code == 404:
```

Comment:
Should it be `elif` here?

Reply:
logically it should be no difference right?

Reply:
Yes logically it makes no difference, but since the two cases are mutually exclusive, it makes it clearer that only one of them will execute.